spring-cloud-dataflow-server-core/src/main/resources/schemas/sqlserver/V4_1-dataflow.sql

```diff
@@ -0,0 +1,3 @@
+-- old table should be dropped if exists
+-- drop table TASK_SEQ;
```

Comment:
May want to add a note that states, " Set start with a value of 1 greater than current value in TASK_SEQ table, if TASK_SEQ table has value greater than 1"